### PR TITLE
All/refactor signers

### PR DESCRIPTION
**Pull request by FrozenKiwi**
Rename "@thecointech/accounts" => "@thecointech/signers"
Extracting basic account definitions into new library "@thecointech/accounts".

The new library is responsible for defining & storing accounts.  By removing these features from shared, we can now easily mock this library to give simple control for defining initial accounts on page load.

Mocking the account load process for dev:live was a bit more complicated due to it's async nature.  Fixed by enabling top-level await in webpack

### CODE REVIEW

File: __mocks__/@thecointech/account/store_devlive.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const address = NormalizeAddress(await signer.getAddress());
+  signer.address = address;
+  signer._isSigner = true;
+
+  _devWallets[address] = buildNewAccount(name, signer);
+  console.log('Loaded remote account: ' + address);
+  if (active) { _initial = address }
+}
+
+await addRemoteAccount('client1', true);
```
Comment (FrozenKiwi) [inline __mocks__/@thecointech/account/store_devlive.js:19]: Woo! Top-level await :-)

File: apps/site-app/src/containers/App/index.tsx [hunk 2]
```diff
@@ -28,24 +27,12 @@ import '../../semantic/semantic.css';
...
 
-let hasRun = false;
-
 export const App = () => {
   useFxRatesStore();
   useSidebar();
   const location = useLocation();
   const contextRef = createRef<HTMLDivElement>();
 
-  if (process.env.CONFIG_NAME === 'devlive') {
```
Comment (FrozenKiwi) [inline apps/site-app/src/containers/App/index.tsx:39]: Our code shouldn't contain configuration switches like this.

By mocking accounts, we put all of our configuration switching in one place

File: apps/site-app/src/reducers.ts [hunk 3]
```diff
@@ -21,14 +21,10 @@ function createReducer(injectedReducers?: ReducersMapObject): Reducer {
...
-  if (process.env.NODE_ENV === 'development') {
-    const { getDevWallets } = require('@thecointech/accounts');
-    return getDevWallets();
-  }
+  console.log('loading initial accounts');
   return {
-    active: null,
-    map: readAllAccounts()
+    active: getInitialAddress(),
+    map: getAllAccounts()
```
Comment (FrozenKiwi) [inline apps/site-app/src/reducers.ts:27]: All environments now load accounts through the same path

File: apps/site-nft/src/components/App/index.tsx [hunk 4]
```diff
@@ -24,16 +23,6 @@ import styles from './styles.module.less';
...
-  if (process.env.NODE_ENV === 'development') {
-    //useDevPrep(account)
-    if (process.env.CONFIG_NAME === 'devlive') {
-      const accountsApi = useAccountStoreApi();
-      React.useEffect(() => {
-        const { addDevLiveAccounts } = require('@thecointech/accounts');
-        addDevLiveAccounts(accountsApi)
-      })
-    }
-  }
```
Comment (FrozenKiwi) [inline apps/site-nft/src/components/App/index.tsx:36]: way too much duplicate code between NFT & App still

File: libs/site-base/internal/webpack/webpack.dev.babel.js [hunk 5]
```diff
@@ -9,97 +9,114 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
...
     }
   }
 }
+
+function getDevLiveOptions() {
+  return {
+    experiments: {
+      topLevelAwait: true,
+    },
+    // Re-direct store to js file
```
Comment (FrozenKiwi) [inline libs/site-base/internal/webpack/webpack.dev.babel.js:109]: we don't actually mock devlive, we just redirect queries to account/store to our devlive-specific version

File: apps/site-nft/src/components/AddAccount/Store/Providers/Google/StoreGoogle.tsx [hunk 6]
```diff
@@ -157,7 +156,7 @@ function fetchAndVerifyWallet(address: string) {
     alert("Cannot upload this wallet: it is not a local account");
     throw new Error("Could not find local account: " + address);
   }
-  if (wallet.privateKey) {
+  else if (wallet.privateKey) {
     alert("Could upload decrypted wallet - if you are seeing this, contact support@thecoin.io");
```
Comment (MarieTheCoin) [inline apps/site-nft/src/components/AddAccount/Store/Providers/Google/StoreGoogle.tsx:160]: We should take the time to change that in a translatable message.

**Review by MarieTheCoin — APPROVED** [review]

File: apps/site-nft/src/components/AddAccount/Store/Providers/Google/StoreGoogle.tsx (hunk 6 above)
Comment (FrozenKiwi) [reply apps/site-nft/src/components/AddAccount/Store/Providers/Google/StoreGoogle.tsx:160]: Issue created to remind me.  It should be done, but it's not central to this PR (and I need these PR's in tonight so can setup testing thingy tomorrow)

**Pull request merged**